### ISSUE TITLE
Validate fields on plan screen before starting scenario

### ISSUE THREAD

**Issue by damies13** [damies13/rfswarm#126]
**Is your feature request related to a problem? Please describe.**
A few people have reported that rfswarm doesn't run anything when they miss selecting a test case from dropdown in the plan screen i've seen this in the discord chat and here in Issue #124

**Describe the solution you'd like**
Perform a validation of the fields on the plan screen when use clicks play before starting scenario, if a field is missed a dialogue should be presented to mention whats missing and test not started.

If possible also put a red ring around the fields that a missing data.

**Comment by damies13** [comment]
Fix merged into v1.1.0 Release Branch

Example of the validation messages below:

![Screenshot from 2023-03-06 18-51-24](https://user-images.githubusercontent.com/15232871/223061962-1dd1238b-276b-4aad-a094-a2ca3cd3ebcd.png)
